opencsp/test/data/sofast_common/Facet_split_NSTTF.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
ditto

Reply:
Same comment as above. These specific JSON files define the physical shape of the facet being measured. But if there is more opportunity to use randomness in these tests, I could implement it.